### ISSUE TITLE
[BUG] Panic when json field is null

### ISSUE THREAD

**Issue by jedvardsson** [nao1215/sqly#42]
A panic is thrown when a json field is null.

**To Reproduce**
1. user.json
```json
[
  {
    "user_id": "1",
    "first_name": null,
    "last_name": null,
  }
]
```

2. `sqly --sql "select * from user limit 1" --csv user.json`
```
goroutine 1 [running]:
github.com/nao1215/sqly/domain/model.(*JSON).ToTable(0x1400010aed0)
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/domain/model/json.go:32 +0x5d8
github.com/nao1215/sqly/shell.CommandList.importCommand(0x5?, 0x1400014c8c0, {0x1400007e280?, 0x14000026960?, 0x14000026950?})
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/shell/import.go:45 +0x50c
github.com/nao1215/sqly/shell.(*Shell).init(0x1400014c8c0)
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/shell/shell.go:128 +0xb0
github.com/nao1215/sqly/shell.(*Shell).Run(0x1400014c8c0)
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/shell/shell.go:80 +0x34
main.run({0x140000120f0?, 0x1049cd220?, 0x104d9b068?})
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/main.go:30 +0xb4
main.main()
	/Users/bob/go/pkg/mod/github.com/nao1215/sqly@v0.5.2/main.go:16 +0x34
```	

**Expected behavior**
one row of csv data
```
user_id,person_name
1,,
```

**Desktop (please complete the following information):**
 - macOS Sonoma 14.4.1 (23E224)
 - go version go1.22.2 darwin/arm64
 - sqly v0.5.2

**Additional context**

**Comment by nao1215** [comment]
@jedvardsson
Thanks for writing two bug reports. It takes time to fix bugs.

**Comment by nao1215** [comment]
Fix this bug at PR #46.
